larray/inout/excel.py

```diff
@@ -178,6 +178,17 @@ def read_excel(filepath, sheet=0, nb_axes=None, index_col=None, fill_value=np.na
     geo\\time      2013      2014      2015
      Belgium  11137974  11180840  11237274
       France  65600350  65942267  66456279
+
+    Extract array from a given range (useful when several arrays are stored in the same sheet)
```

Comment:
There are other cases where this could be useful (e.g. if there is some text before, after or next to the data you want to load), so I am unsure the parentheses add anything.